var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -104,13 +98,23 @@ def cmake_args(self):
         else:
             options.append('-DGMX_GPU:BOOL=OFF')
 
-        simd_value = self.spec.variants['simd'].value
-        if simd_value == 'auto':
-            pass
-        elif simd_value == 'none':
+        simd_features = {
+            'sse2': 'SSE2',
+            'sse4_1': 'SSE4.1',
+            'avx128fma': 'AVX_128_FMA',
+            'avx': 'AVX_256',
+            'axv128': 'AVX2_128',
```

Comment:
We don't have `avx128` and `avx128fma` features in `microarchitectures.json`, at least not right now. It seems to me that the guide [here](http://manual.gromacs.org/documentation/2018/install-guide/index.html#simd-support) ties those flags to specific targets. Should we consider using something like:
```python
if self.spec.target > llnl.util.cpu.targets['bulldozer']:
   # Append the appropriate flag
```
?